Tests/SourceKittenFrameworkTests/StringTests.swift

```diff
@@ -228,6 +228,15 @@ class StringTests: XCTestCase {
         // no crash on bad input
         XCTAssertEqual("ab\\".unescaped, "ab")
     }
+
+    func testResponseFiles() {
+        let responseContents = "3\n4\n5\n"
```

Comment:
Maybe have a "file" with a space in its path?
```suggestion
        let responseContents = "3 4\n5\n"
```

Might want to separate the test string with some other character than space in that case too (`,` or `|`) for example.